src/pages/HttpBin/index.js

```diff
@@ -98,13 +95,13 @@ const HttpBin = () => {
         </Grid>
         <Grid item xs={8}>
           <FormInput
-            label="Query parameters "
-            helperText="Query parameters."
+            label="URL"
+            helperText="Use '/' to set empty path"
             multiline={false}
-            value={queryParameters}
+            value={urlFormValue}
             rowsMax={1}
             onChange={(event) => {
-              setQueryParameters(event.target.value);
+              setURLlFormValue(event.target.value);
             }}
           />
```

Comment:
the main question, do you really need this input?

fe: can I send a POST request to `/ip` endpoint ?

The main idea of this task is getting ability to send any data with any body and query parameters.
According to the task, I don't see any point that describes URL.

So, you don't need this input field, do you?

Reply:
what do you mean about sent POST to `/ip` endpoint?

Reply:
according to your behaviour, I can see, that I can send any request to any URI. Is it correct?
according to the task description, I can't see any line that describes sending requests to any URI. Right?

So, you don't need this input field, do you?

Reply:
Ok i will delete. Update in next commit